rasgotransforms/rasgotransforms/render/environment.py

```diff
@@ -44,6 +44,7 @@ def rasgo_globals(self):
             "cleanse_name": cleanse_template_symbol,
             "raise_exception": raise_exception,
             "itertools": {"combinations": combinations, "permutations": permutations, "product": product},
+            "dw_type": lambda: self.dw_type.value,
```

Comment:
why the lambda here?

Reply:
This is the list of globals/functions that are available in the jinja environment, so this makes `dw_type()` available in any over our transforms to get the dw_type. There is an example of this in the pr description.